TheMealDB/TheMealDB/Moduls/CategroryScreen/Controller/CattegroryScreenController.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  CattegroryScreenController.swift
+//  TheMealDB
+//
+//  Created by mac on 12/18/18.
+//  Copyright © 2018 mac. All rights reserved.
+//
+
+import UIKit
+
+class CattegroryScreenController: UIViewController {
+    @IBOutlet weak var tableView: UITableView!
+    
+    private var categoryArray = [CategroryItem]() {
+        didSet {
+            self.tableView.reloadData()
+        }
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.dataSource = self
+        tableView.delegate = self
+        register()
+        loadData()
+    }
+    
+    private func register() {
```

Comment:
đặt tên method cần đúng với nhiệm vụ của nó
```suggestion
    private func initTableView() {
```
method này cần bao gồm cả đoạn codeowr dòng 22 và 23